packages/server/src/fhir/operations/utils/cms1500pdf.ts

```diff
@@ -0,0 +1,406 @@
+  const otherInsured = otherCoverage?.subscriber ? await repo.readReference(otherCoverage.subscriber) : undefined;
+  const referralRequest = claim.referral ? await repo.readReference(claim.referral) : undefined;
+  const referrer = referralRequest?.requester ? await repo.readReference(referralRequest.requester) : undefined;
+
+  const insuranceType = coverage.type?.coding?.[0].code ?? coverage.type?.coding?.[0].display ?? '';
+
+  const relationship = (insured as RelatedPerson | undefined)?.relationship?.[0]?.coding?.[0].code ?? 'self';
+
+  const relatedToEmployment =
+    claim.supportingInfo?.some((info) => info.category.coding?.[0].code === 'employmentimpacted') ?? false;
+  const relatedToAutoAccident = claim.accident?.type?.coding?.some((code) => code.code === 'MVA') ?? false;
+
+  const accidentLocationState = claim.accident?.locationAddress?.state;
+  const relatedToOtherAccident = !relatedToAutoAccident && !!claim.accident;
+
+  const dateOfCurrentIllness = claim.supportingInfo?.find(
+    (info) => info.category.coding?.[0].code === 'onset'
+  )?.timingDate;
+
+  const employmentImpacted = claim.supportingInfo?.find(
+    (info) => info.category.coding?.[0].code === 'employmentimpacted'
+  );
+
+  const hospitalization = claim.supportingInfo?.find((info) => info.category.coding?.[0].code === 'hospitalized');
+
+  const priorAuthRefNumber = claim.insurance[0]?.preAuthRef?.[0];
+
+  const outsideLab = claim.supportingInfo?.find((info) => info.category.coding?.[0].code === 'outsidelab');
+  const outsideLabCharges = outsideLab ? formatQuantity(outsideLab.valueQuantity) : '';
+
+  const diagnosis = (claim.diagnosis ?? [])
+    .map(
+      (d) => d.diagnosisCodeableConcept?.coding?.find((code) => code.system === `${HTTP_HL7_ORG}/fhir/sid/icd-10`)?.code
+    )
+    .filter(Boolean) as string[];
+
+  const resubmissionCode = claim.related?.[0].relationship?.coding?.find((code) => code.code === 'prior')?.display;
+
+  const originalReference = claim.related?.[0].claim?.display;
+
+  const patientAccountNumber = claim.supportingInfo?.find(
+    (info) =>
+      info.category?.coding?.find((code) => code.code === 'info') &&
+      info.code?.coding?.find((code) => code.code === 'patientaccount')
+  )?.valueString;
+
+  const patientPaid = formatQuantity(
+    claim.supportingInfo?.find(
+      (info) =>
+        info.category?.coding?.find((code) => code.code === 'info') &&
+        info.code?.coding?.find((code) => code.code === 'patientpaid')
+    )?.valueQuantity
+  );
+
+  const taxIdentifier = insurer.identifier?.find((id) => id.type?.coding?.find((code) => code.code === 'TAX'));
+
+  const docDefinition: TDocumentDefinitions = {
+    defaultStyle: {
+      font: 'Helvetica',
+      fontSize: 8,
+    },
+    pageSize: { width: PAGE_WIDTH, height: PAGE_HEIGHT },
+    pageMargins: 0,
+    content: [
+      {
+        image: 'data:image/png;base64,' + imageToBase64(__dirname + '/cms1500.png'),
```

Comment:
I think we can cache this in the module scope since this doesn't change for the lifetime of the server, ie.
```ts
const CMS_1500_IMG_DATA_URL = 'data:image/png;base64,' + imageToBase64(__dirname + '/cms1500.png');
```
```suggestion
        image: CMS_1500_IMG_DATA_URL,
```